dev/golangci/lint.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"path"
+	"strings"
+)
+
+const (
+	lintImageRepo   = "docker.io/golangci/golangci-lint"
+	lintImageTag    = "v1.59-alpine"
```

Comment:
We should have a `WithVersion` to be able to override this.

Reply:
I have some concerns about that (no guarantee that every version of the tool works with every version of the module + not sure anyone actually needs to control the version), but I can understand the frustration in case someone does need this urgently and there is no flag for it.

For now can we agree to make it a follow-up improvement, like config file customization?